### PR TITLE
[fix #995] supported extensions list in NoPublicationInfo component

### LINKED ISSUE #995: add audiobook support to the drop-box message
If the bookshelf is empty, the message is currently "Drop EPUB files in this window."

Let's update it to "Drop EPUB or Audiobook files in this window."

### PR DESCRIPTION
**Pull request by panaC**
adds the supported extensions list in NoPublicationInfo component

fix #995

### CODE REVIEW

File: src/resources/locales/en.json [hunk 1]
```diff
@@ -49,9 +49,10 @@
         "lang": "Language",
         "moreInfo": "More Info",
         "myBooks": "My books",
-        "noPublicationHelpL1": "Drop EPUB files in this window",
+        "noPublicationHelpL1": "Drop off the publication",
```
Comment (danielweck) [inline src/resources/locales/en.json:52]: “drop a publication here” probably a better choice of terms.

File: src/renderer/library/components/catalog/NoPublicationInfo.tsx [hunk 2]
```diff
@@ -35,6 +36,16 @@ class NoPublicationInfo extends React.Component<IProps, undefined> {
...
                     <p>{__("catalog.noPublicationHelpL2")}</p>
                     <p>{__("catalog.noPublicationHelpL3")}</p>
+                    <p>{__("catalog.noPublicationHelpL4")}</p>
+                    {
+                        acceptedExtensionArray.map(
+                            (ext) => <span>
+                                {
+                                    ext
+                                }
+                            </span>,
```
Comment (danielweck) [inline src/renderer/library/components/catalog/NoPublicationInfo.tsx:46]: trailing comma? maybe array.join or array.reduce instead of array.map?

Comment (panaC) [reply src/renderer/library/components/catalog/NoPublicationInfo.tsx:46]: it depends on the design

Comment (danielweck) [reply src/renderer/library/components/catalog/NoPublicationInfo.tsx:46]: but surely a trailing comma in the UI cannot be a good thing, right?

Comment (panaC) [reply src/renderer/library/components/catalog/NoPublicationInfo.tsx:46]: @llemeurfr what do you prefer?

Comment (llemeurfr) [reply src/renderer/library/components/catalog/NoPublicationInfo.tsx:46]: I prefer no trailing comma

Comment (danielweck) [reply src/renderer/library/components/catalog/NoPublicationInfo.tsx:46]: Let's fix this small issue, and let's get this merged?

Comment (panaC) [reply src/renderer/library/components/catalog/NoPublicationInfo.tsx:46]: I don't know what do you want ?
there is no trailing comma.
this displays the list of extensions.

Comment (danielweck) [reply src/renderer/library/components/catalog/NoPublicationInfo.tsx:46]: NoComma

You are correct! There is not trailing comma (I wonder what happens to the comma after the closing `span` element?)

Anyway, let's merge then :)

Comment (danielweck) [reply src/renderer/library/components/catalog/NoPublicationInfo.tsx:46]: JSX weirdness. Resolved.

**Pull request merged**